app/models/concerns/foreman_remote_execution/exportable.rb

```diff
@@ -0,0 +1,54 @@
+# This concern makes it easy to export an ActiveRecord object with specified
+# attributes and associations only.  If a specified assocation also includes
+# this concern, then it will likewise be exported. Custom attributes can be
+# specified as an options hast consisting of a custom export lambda.
+#
+# Example:
+#   attr_exportable :name, :address, :company => ->(user) { user.company.name }
+#
+module ForemanRemoteExecution
+  module Exportable
+    extend ActiveSupport::Concern
```

Comment:
This will need to be included here to avoid a cherry-picking nightmare for Satellite, as it's been approved for inclusion in Sat 6.2.  And it's just a nice feature, so we can do at least one more release for 1.11 that has this.

There should be no conflict as our concerns are namespaced if the foreman PR gets merged into 1.12. We can remove the concern from us when we drop 1.11 support.  Sound OK?